### PR TITLE
Add docs and samples for AWS auth

### PR DESCRIPTION
**Pull request by ekoleda-codaio**
A maker I'm working with wants to connect to an AWS service, so adding some docs and samples showing how to use `AWSAccessKey` auth. Assigning to Chris since I think you're most familiar with this auth method.

### CODE REVIEW

**Comment by ekoleda-codaio** [comment]
Friendly ping.

File: docs/guides/basics/authentication/index.md [hunk 1]
```diff
@@ -316,6 +316,28 @@ Enabling the option `shouldAutoAuthSetup: true` further simplifies the sign-in e
 [View Sample Code][sample_coda_api]{ .md-button }
 
 
+### AWS Signautre Version 4
+
+Amazon Web Services (AWS) uses a a proprietary authentication scheme that requires each requests to be signed with a key and secret. Packs support this using the [`AWSAccessKey`][AWSAccessKey] authentication. You must specify which AWS service you are connecting to, since that is a required part of the signature. For example:
```
Comment (chris-codaio) [inline docs/guides/basics/authentication/index.md:321]: AWS supports multiple forms of authentication.  This is one of them.  Might want to clarify the statement here.

**Comment by chris-codaio** [comment]
> Friendly ping.

My apologies.  I didn't realize this was split across two PRs.  Looking now.

**Review by chris-codaio — APPROVED** [review]

File: docs/guides/basics/authentication/index.md (hunk 1 above)
Comment (ekoleda-codaio) [reply docs/guides/basics/authentication/index.md]: Good callout, done.

**Pull request merged**